### ISSUE TITLE
Some german comics

### ISSUE THREAD

**Issue by Matthias84** [tardypad/sailfishos-daily-comics#24]
Hi, I like to suggest adding a few german comicstrips:

```
m_info.id             = QString("islieb");
m_info.name           = QString("is lieb?");
m_info.authors        = QStringList("Eylou");
m_info.homepage       = QUrl("http://islieb.de");
m_info.country        = QLocale::Germany;
m_info.language       = QLocale::German;
m_info.startDate      = QDate::fromString("2014-02", "yyyy-MM");
m_info.endDate        = QDate::currentDate();

m_info.id             = QString("nichtlustig");
m_info.name           = QString("nicht lustig");
m_info.authors        = QStringList("Joscha Sauer");
m_info.homepage       = QUrl("http://nichtlustig.de");
m_info.country        = QLocale::Germany;
m_info.language       = QLocale::German;
m_info.startDate      = QDate::fromString("2000-01", "yyyy-MM");
m_info.endDate        = QDate::currentDate();

m_info.id             = QString("erzählmirnix");
m_info.name           = QString("erzähl mir nix!");
m_info.authors        = QStringList("Nadja Hermann");
m_info.homepage       = QUrl("http://erzaehlmirnix.wordpress.com");
m_info.country        = QLocale::Germany;
m_info.language       = QLocale::German;
m_info.startDate      = QDate::fromString("2012-05", "yyyy-MM");
m_info.endDate        = QDate::currentDate();
```

Sorry but I'm not experienced enough to contribute the scraper methods itself :-/

**Comment by tardypad** [comment]
Available now in version 1.5.1
Enjoy!

**Comment by Matthias84** [comment]
Thanks  / Danke @tardypad :-)
